### PR TITLE
Add release post with details

### PR DESCRIPTION
**Pull request by 197g**

### CODE REVIEW

File: _posts/2019-10-04-release-0.22.3-and-fuzzit-dev.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+author: HeroicKatora
+title: "Fuzzing and image reading in 0.22.3 and onwards"
+---
+
+Many use cases for decoding pixel matrices involve an inherently untrustworthy
+environment. Think of a webbrowser handling media sent by potentially shady
+servers or an open image processing service in the web.
+
+`image` offers a neat yet powerful and versatile core of operations. And for
+that reason it is used in a game libraries, graphics API bindings and, as the
```
Comment (JohnTitor) [inline _posts/2019-10-04-release-0.22.3-and-fuzzit-dev.md]:
```suggestion
that reason it is used in game libraries, graphics API bindings and, as the
```

File: _posts/2019-10-04-release-0.22.3-and-fuzzit-dev.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+author: HeroicKatora
+title: "Fuzzing and image reading in 0.22.3 and onwards"
+---
+
+Many use cases for decoding pixel matrices involve an inherently untrustworthy
+environment. Think of a webbrowser handling media sent by potentially shady
+servers or an open image processing service in the web.
```
Comment (JohnTitor) [inline _posts/2019-10-04-release-0.22.3-and-fuzzit-dev.md]:
```suggestion
servers or an open image processing service on the web.
```

File: _posts/2019-10-04-release-0.22.3-and-fuzzit-dev.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+[cargo-fuzz]: https://github.com/rust-fuzz/cargo-fuzz
+[fuzzit]: https://fuzzit.dev/
+
+## A new configurable reader interface
+
+So, how else can the newly released version `0.22.3` help?
+
+If you look at the bigger picture then the stability goals for trustless
+computation also help all other areas as well. Accidents, misconfiguration and
+data corruption happens more often in the real world than we'd like. Too many
```
Comment (JohnTitor) [inline _posts/2019-10-04-release-0.22.3-and-fuzzit-dev.md]:
```suggestion
data corruption happen more often in the real world than we'd like. Too many
```

File: _posts/2019-10-04-release-0.22.3-and-fuzzit-dev.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+hours of work were lost to crashing editors (I've heard both Photoshop and GIMP
+cited as prime examples), the root cause being involutnary resource exhaustion.
```
Comment (JohnTitor) [inline _posts/2019-10-04-release-0.22.3-and-fuzzit-dev.md]:
```suggestion
cited as prime examples), the root cause being involuntary resource exhaustion.
```

**Review by JohnTitor — APPROVED** [review]
some nits

**Comment by 197g** [comment]
@fintelia I'll be gone for the weekend so feel free to merge if you think it is ready. Or make adjustments on the branch.

**Comment by fintelia** [comment]
The content seems good. If you want, I can make an edit pass to streamline the writing

**Comment by 197g** [comment]
Sure. I'm not available for feedback but you can surely solicit someone else if need be. Signing off for the weekend.

**Comment by 197g** [comment]
@fintelia I'm back, anything you wanted to add?

**Comment by fintelia** [comment]
I just committed some changes. At this point I think it is ready to post

**Review by fintelia — APPROVED** [review]

**Comment by 197g** [comment]
Yep, definitely an improvement. Thank you for reviewing!

**Pull request merged**